recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -225,5 +225,18 @@ def package_info(self):
         manifest_subfolder = "bin" if self.settings.os == "Windows" else os.path.join("res", "vulkan", "explicit_layer.d")
         vk_layer_path = os.path.join(self.package_folder, manifest_subfolder)
         self.runenv_info.prepend_path("VK_LAYER_PATH", vk_layer_path)
+
+        # Update runtime discovery paths to allow libVkLayer_khronos_validation.{so,dll,dylib} to be discovered
+        # and loaded by vulkan-loader when the consumer executes
+        # This is necessary because this package exports a static lib to link against and a dynamic lib to load at runtime
+        runtime_lib_discovery_path = "LD_LIBRARY_PATH"
+        if self.settings.os == "Windows":
+            runtime_lib_discovery_path = "PATH"
+        if self.settings.os == "Macos":
+            runtime_lib_discovery_path = "DYLD_LIBRARY_PATH"
+        for libdir in [os.path.join(self.package_folder, libdir) for libdir in self.cpp_info.libdirs]:
+            self.output.info(f"Prepending {libdir} to {runtime_lib_discovery_path}")
```

Comment:
```suggestion
```

I would remove these messages, it can be called many times. It should be the job of VirtualRunEnv to display something on demand.